parlai/zoo/model_list.py

```diff
@@ -1872,4 +1872,175 @@
+        "id": "blenderbot2",
+        "path": "zoo:blenderbot2/memory_decoder/model",
+        "agent": "bart",
+        "task": "multi",
+        "project": "https://parl.ai/projects/blenderbot2/",
+        "description": (
+            "The memory decoder for BlenderBot2. Either generates a memory "
+            "to write or a token indicating no memory can be written."
+        ),
+        "example": ("parlai interactive -mf zoo:blenderbot2/memory_decoder/model"),
+        "result": (
+            "Enter Your Message: i love reading; harry potter was such a good series\n"
+            "[Bart]: I love reading. I like the Harry Potter series."
+        ),
+    },
+    {
+        "title": "BlenderBot2 3B",
+        "id": "blenderbot2",
+        "path": "zoo:blenderbot2/blenderbot2_3B/model",
+        "agent": "projects.blenderbot2.agents.blenderbot2:BlenderBot2FidAgent",
+        "task": "wizard_of_internet",
+        "project": "https://parl.ai/projects/blenderbot2/",
+        "description": ("BlenderBot2 3B Model. See project for details."),
+        "example": (
+            "parlai interactive -mf zoo:blenderbot2/blenderbot2_3B/model --search-server relevant_search_server"
+        ),
+        "result": (
+            "Enter Your Message: my favorite tv show is wandavision\n"
+            "[BlenderBot2Fid]: Who is your favorite character in WandaVision? Mine is Elizabeth Olsen."
+        ),
+    },
+    {
+        "title": "BlenderBot2 400M",
+        "id": "blenderbot2",
+        "path": "zoo:blenderbot2/blenderbot2_400M/model",
+        "agent": "projects.blenderbot2.agents.blenderbot2:BlenderBot2FidAgent",
+        "task": "wizard_of_internet",
+        "project": "https://parl.ai/projects/blenderbot2/",
+        "description": ("BlenderBot2 400M Model. See project for details."),
+        "example": (
+            "parlai interactive -mf zoo:blenderbot2/blenderbot2_400M/model --search-server relevant_search_server"
+        ),
+        "result": (
+            "Enter Your Message: my favorite food is chicken parmesan, do you have a good recipe?\n"
+            "[BlenderBot2Fid]: I don't have a recipe, but I do know how to make it at home. It's easy to make."
+        ),
+    },
+    {
+        "title": "Bart Base Wizard of Internet",
+        "id": "sea",
+        "path": "zoo:sea/bart_base/model",
+        "agent": "bart",
+        "task": "wizard_of_internet",
+        "project": "https://parl.ai/projects/sea/",
+        "description": ("BART-Large 400m model trained on Wizard of Internet."),
+        "example": ("parlai interactive -mf zoo:sea/bart_base/model"),
+        "result": (
+            "Enter Your Message: Do you know about the world cup 2022?\n"
+            "[Bart]: I heard that the fifa games are coming back."
+        ),
+        "example2": (
+            "parlai eval_model -mf zoo:sea/bart_base/model -t wizard_of_internet --num-examples 100"
+        ),
+        "result2": ("TODO"),
```

Comment:
you can add this here if you have it but not a huge deal if not @mojtaba-komeili